tools/gff2JSON/gff-to-json.xml

```diff
@@ -0,0 +1,61 @@
+<tool id="gff-to-JSON_wrapper" name="GFF to JSON" version="0.1.1">
+    <description>
+        Generate a JSON formated Gene file from GFF format
+    </description>
```

Comment:
Please add:

```
    <requirements>
        <requirement type="package" version="2.90">perl_json</requirement>
    </requirements>
```

I will later add a `tool_dependencies.xml` file to install the [JSON](http://search.cpan.org/~makamaka/JSON-2.90/lib/JSON.pm) Perl module.